pyproject.toml

```diff
@@ -0,0 +1,96 @@
+[tool.black]
```

Comment:
File shouldn't be part of this PR

Reply:
You're right... forgot to put into gitignore